esrally/utils/process.py

```diff
@@ -229,3 +229,35 @@ def rally_process(p: psutil.Process) -> bool:
         )
 
     kill_all(rally_process)
+
+
+def current_pid() -> int:
```

Comment:
Why do we need to wrap os.getpid?

Reply:
Heh, this was left over from when I was using `psutil` for the purpose, but realized this was a much faster call.